metalearn/metafeatures/resources.py

```diff
@@ -1,10 +1,71 @@
-"""
-Exposes all the package's bundled data files as file path strings.
-Needed to allow the data files to be successfully accessed across platforms 
-and install types.
-"""
-
+from typing import List
```

Comment:
I'm kinda confused about this file, it seems to be mixing two different kinds of resources (i.e. files required to be downloaded with the package vs general arguments to metafeature computers).

Reply:
I think you're right @macetheace96, the `resources.py` file originally just exported the module's static assets, including the `metafeatures.json` file. Since the `ResourceComputerMap` replaces `metafeatures.json`, it made sense to export that here. Now though, static assets and code are mixed, so I've broken `METAFEATURES_JSON_SCHEMA` out into a `static_assets.py` module in a recent commit to address this issue.